tests/integration/catalog-source.yaml

```diff
@@ -11,5 +11,19 @@ spec:
       maintainer: weaveworks (https://github.com/weaveworks/profiles)
       prerequisites:
         - Kubernetes 1.18+
+    - name: multi-version-weaveworks-nginx
+      description: This installs nginx.
+      version: v0.1.0
+      url: https://github.com/weaveworks/profiles-examples
+      maintainer: weaveworks (https://github.com/weaveworks/profiles)
+      prerequisites:
+        - Kubernetes 1.18+
+    - name: multi-version-weaveworks-nginx
+      description: This installs nginx.
+      version: v0.1.1
+      url: https://github.com/weaveworks/profiles-examples
+      maintainer: weaveworks (https://github.com/weaveworks/profiles)
+      prerequisites:
+        - Kubernetes 1.18+
```

Comment:
shall we also update the `example/profile-source-catalog.yaml` and add a `weaveworks-nginx` `v0.1.1` too? (and create that actual tag, so we don't run into problems :wink: )

Reply:
https://github.com/weaveworks/profiles-examples/blob/update-available/weaveworks-nginx/profile.yaml

yeah I've already did something there. :D

Reply:
need to merge that and cut a tag :smile:

Reply:
makes sense :)